guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>Once the explanatory variables are preprocessed this is now used as input for the prepare_tabulardata method from the tabular learner in the arcgis.learn. </p><p> </p><p>I think saying "preprocessed" and "tabular learner" is causing some confusion, we should rephrase it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>